### ISSUE TITLE
[0.9] Make the Steel / Modern pickaxe hit faster

### ISSUE THREAD

**Issue by ThePiachu** [StrangeLoopGames/EcoSuggestions#1075]
During the recent playthrough on the test server we got to the modern tools a bit earlier than usual due to people taking Industry / Electronics / Oil Drilling in parallel. While a lot of the modern tools are great, the pickaxe feels a bit underwhelming if you're not digging harder rocks (which will be the case for a lot of iron miners). When transitioning from the Iron to Steel and from Steel to Modern while working in the iron mines I thought to myself - "these feel just like the previous tier", which is not a great feeling to have with tool upgrades. It would be nice if the higher-tier pickaxes allowed you to swing faster in addition to hitting harder, making them more useful for all miners.

**Comment by trickenso** [comment]
I pointed this out in another issue I cannot find: if you're mining sandstone, all the picks function the same.
Could they maybe break multiple blocks if they're working in soft stone, or similar? This might be more in line with the upgrades to the hoe, for instance. Noting that it might also make it harder to control - some kind of predictable shape of blocks being broken would be relatively easy to learn.

Is the shovel is also a candidate for this kind of thinking? Maybe uprooting older trees or an area stamp to dig multiple blocks would also feel more like an upgrade.
